docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/google-tag-manager-custom-template/v3-tags/ecommerce-tag-template/ecommerce-tag-configuration/index.md

```diff
@@ -0,0 +1,98 @@
+In this textbox you can specify the argument to the ecommerce function. This can be a Variable that evaluates to a corresponding object.
+
+#### Additional Tracking Parameters
+
+**Add Custom Context Entities**
+
+Use this table to attach [custom context entities](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/index.md#custom-context) to the Snowplow event. Each row can be set to a Google Tag Manager variable that returns an array of custom contexts to add to the event hit.
+
+**Set Custom Timestamp**
+
+Set this to a UNIX timestamp in case you want to [override the default timestamp](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/index.md#setting-the-true-timestamp) used by Snowplow.
+
+  </TabItem>
+  <TabItem value="ga4" label="GA4 Ecommerce">
+
+![](images/02_ga4_tracking_parameters.png)
+
+#### GA4 Ecommerce Function
+
+In this section you can select the [Google Analytics 4 Ecommerce function](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/snowplow-ecommerce/index.md#google-analytics-4-ecommerce) to use.
+
+#### GA4 Ecommerce Arguments
+
+**DataLayer ecommerce**
+
+Here you can specify the dataLayer ecommerce variable to use.
+
+**Options object**
+
+Here you can specify a variable returning an object holding additional information for the ecommerce event (e.g.including currency, finalCartValue, step etc).
+
+  </TabItem>
+  <TabItem value="ua" label="Universal Analytics Enhanced Ecommerce">
+
+![](images/02_ua_tracking_parameters.png)
+
+#### Universal Analytics Enhanced Ecommerce Function
+
+In this section you can select the [Universal Analytics Enhanced Ecommerce function](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/snowplow-ecommerce/index.md#universal-analytics-enhanced-ecommerce) to use.
+
+#### Universal Analytics Enhanced Ecommerce Arguments
+
+**DataLayer ecommerce**
+
+Here you can specify the dataLayer ecommerce variable to use.
+
+**Options object**
+
+Here you can specify a variable returning an object holding additional information for the ecommerce event (e.g.including currency, finalCartValue, step etc).
+
+  </TabItem>
+</Tabs>
+
+## Snowplow Tracker and Ecommerce Plugin Settings
+
+![](images/04_tracker_plugin_settings.png)
+
+### Tracker Settings
+
+The Snowplow v3 Ecommerce tag template **requires** a Snowplow v3 Settings Variable to be setup. In this section you can select the Google Tag Manager variable of type [Snowplow v3 Settings](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/google-tag-manager-custom-template/v3-settings-variable/index.md) to use.
+
+### Plugin Settings
+
+In this section you can select how the plugin will be added. The available options are:
+
+- **jsDelivr**: To get the plugin URL from jsDelivr CDN
+- **unpkg**: To get the plugin URL from unpkg CDN
```

Comment:
Do jsDelivr/unpkg let you set the plugin version, or this pulled from the Settings to keep it in sync with the tracker SDK version?

Reply:
At the moment the version is not pulled from settings. Once you specify one of those 2 options, you then specify the version.

Reply:
Should we note/recommend using a Variable to store your version and re-use it here to keep the plugin and SDK versions in sync? (Though this might bite us if we stop doing versioning in lock-step as has been discussed. :thinking:  Thoughts @igneel64 ?)

Reply:
Created an issue https://github.com/snowplow/snowplow-gtm-tag-template-ecommerce-v3/issues/4 to discuss this for next release.